src/app/components/client/ExposureCard.tsx

```diff
@@ -44,15 +44,44 @@ export type ExposureCardProps = {
 };
 
 export const ExposureCard = ({ exposureData, ...props }: ExposureCardProps) => {
-  return isScanResult(exposureData) ? (
-    <ScanResultCard {...props} scanResult={exposureData} />
-  ) : (
-    <SubscriberBreachCard {...props} subscriberBreach={exposureData} />
-  );
+  const [dataBrokerImage, setDataBrokerImage] = useState<ReactNode>();
+
+  useEffect(() => {
+    const loadDataBrokerImage = async () => {
+      if (isScanResult(exposureData)) {
+        const cleanedName = exposureData.data_broker.split(".")[0];
+        try {
+          const logo = await import(
+            `../client/assets/data-brokers/${cleanedName}.jpg`
+          );
+          setDataBrokerImage(<Image src={logo.default} alt="" />);
+        } catch (error) {
+          // Default to circle logos if logo is not found
+          setDataBrokerImage(<FallbackLogo name={cleanedName} />);
+        }
+      }
+    };
+
+    loadDataBrokerImage().catch((error) => {
+      console.error("Error loading logo:", error);
```

Comment:
Q: Where does this `console.error()` go?
Should we log it to Sentry?

Reply:
It goes to the GCP logs - if it's something we expect might happen occasionally (and there's not much need to investigate and have a stack trace etc) then going through Sentry isn't necessary.

To send to Sentry, you could use `captureException(error)` from Sentry module. Sentry also catches unhandled exceptions, but that means an error page will be displayed to the user, so if we have a fallback in place then that's likely not what we want.